opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -227,7 +227,19 @@ namespace Opm {
 
         using namespace std::placeholders;
         const auto handler = std::bind( handleKW, _1, schedule, props, n_xyz );
+
+#ifdef _MSC_VER
+        for (auto& x : section)
+        {
+            auto keywords = handler(x);
+            for (auto& keyword : keywords)
+            {
+                this->keywords.push_back(keyword);
+            }
+        }
+#else
         this->keywords = fun::concat( fun::map( handler, section ) );
```

Comment:
>> if we were talking lisp or scheme, I'd agree. But in C++, these are not standard constructs, and people are not used to the functional programming paradigm...

> This is just not true.

This statement is quite bold... While awareness of functional programming is on the rise among C++ programmers (and in general) I think it is fair to say that most don't know it very well. At least not well enough to make the use of fun::map and fun::concat easier to understand than the "manual" alternative.

I this particular case I guess I'd (somewhat regretfully) use the more primitive code, perhaps adding a comment that it works around a compiler bug.